modules/varbase_features/varbase_landing/varbase_landing.module

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/**
+ * @file
+ * Contains varbase_media.module.
+ */
+
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function varbase_landing_form_alter(&$form, FormStateInterface $form_state, $form_id) {
```

Comment:
Why not varbase_landing_form_node_landing_page_edit_form_alter

And node_landing_page_form!!